packages/ses/README.md

```diff
@@ -726,11 +726,42 @@ instructions in [SECURITY.md][] to report security-sensitive bugs privately.
 
 For non-security bugs, please use the [regular Issues page][SES Issues].
 
+## Ecosystem Compatibility
+
+Most ordinary JavaScript can run without issues in a realm locked down by SES.
+Exceptions are tracked at [issue #576][incompatibility tracking], and almost
+always take the form of assignments that fail because the
+"[override mistake][override mistake]" prevents overriding properties inherited
+from a frozen intrinsic object in the prototype chain. When that is the case,
+the code is often incompatible with *all* environments in which intrinsic
+objects are frozen (such as in Node.js with the
+[`--frozen-intrinsics`][Node frozen intrinsics] option) and can be fixed by
+replacing `<lhs>.<propertyKey> = <rhs>;` or `<lhs>[<propertyKey>] = <rhs>;` with
+```js
+Object.defineProperties(<lhs>, {
+  [<propertyKey>]: {
+    value: <rhs>,
+    writable: true,
+    enumerable: true,
+    configurable: true,
+  },
+});
+```
+
+Upon encountering an incompatibility, we recommend that you add a comment to
+[issue #576][incompatibility tracking] and file an issue with the external project referencing this section.
```

Comment:
we could reduce the load for filing the issue by providing a template. e.g. 
> Some applications defend themselves by freezing their intrinsics. This project has some assignments or mutations that break in such an environment.
> Specifically,
> [link to source in the project]
>
> Please consider making the project compatible with these environments by avoiding mutations to intrinsics.
> If you don't have the capacity but would accept a PR, please comment to that effect so that a volunteer knows their efforts would be welcomed.

Reply:
Good idea, and thank you for the starting point. PTAL.

Reply:
I think we can narrow this from "assignment or mutation" to just "assignment".

> by avoiding mutations to intrinsics

Is sometimes the issue and should be mentioned when relevant. But by far the most common case is using assignment to override a property inherited from a frozen intrinsic, i.e., the override mistake. We should always lead with the override mistake.

Reply:
And for the override mistake, always first suggest doing the mutation by `defineProperty` rather than assignment, because that is much more likely to be correctness preserving and least effort.

Reply:
Or rather `defineProperties` since that's what is already shown in this PR's example code.